src/App.tsx

```diff
@@ -1,109 +1,33 @@
 import React from 'react';
 
 import './App.scss';
-
-// import postsFromServer from './api/posts';
-// import commentsFromServer from './api/comments';
-// import usersFromServer from './api/users';
+import postsFromServer from './api/posts';
+import commentsFromServer from './api/comments';
+import usersFromServer from './api/users';
+import { Comment } from './types/Comment';
+import { User } from './types/User';
+import { Post } from './types/Post';
+import { PostList } from './components/PostList';
+
+const getCommentsById: (postId: number) =>
+Comment[] = postId => commentsFromServer
+  .filter(comment => comment.postId === postId);
```

Comment:
```suggestion
const getCommentsById = (
  postId: number
): Comment[] => commentsFromServer
  .filter(comment => comment.postId === postId);
```

Reply:
no need to add type of all function, better just to add types for incoming parameters and returned value, that way is shorter and more understandable